entity-framework/core/managing-schemas/migrations/applying.md

```diff
@@ -156,6 +159,76 @@ Note that this can be used to roll back to an earlier migration as well.
 
 For more information on applying migrations via the command-line tools, see the [EF Core tools reference](xref:core/cli/index).
 
+## Bundles
+
+> [!NOTE]
+> This feature was added in EF Core 6.0.
+
+Migration bundles are single-file executables than can be used to apply migrations to a database. They address some of the shortcomings of the SQL script and command-line tools:
+
+* Executing SQL scripts requires additional tools.
+* The transaction handling and continue-on-error behavior of these tools are inconsistent and sometimes unexpected.
```

Comment:
```suggestion
* The transaction handling and continue-on-error behavior of these tools are inconsistent and sometimes unexpected. This can leave your database in an undefined state if a failure occurs when applying migrations.
```